mergify_cli/tests/utils.py

```diff
@@ -0,0 +1,29 @@
+#
+#  Copyright © 2021-2023 Mergify SAS
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Why is the license block needed?

Reply:
It's good practice in general in case a file would be copied alone.
Not strongly needed but good to have 🤷